src/libraries/System.Private.Xml/src/System/Xml/Xslt/XsltContext.cs

```diff
@@ -11,7 +12,7 @@ public interface IXsltContextFunction
         int Minargs { get; }
         int Maxargs { get; }
         XPathResultType ReturnType { get; }
-        XPathResultType[] ArgTypes { get; }
+        XPathResultType[]? ArgTypes { get; }
```

Comment:
Should this really be nullable?
1. in all of our implementation we pass it as non-nullable  (see types derived form `XsltFunctionImpl`).
2. we don't really handle it as nullable on the only place that this is referenced.
https://github.com/dotnet/runtime/blob/3174da7f9612bc6c46762fb73bba7d1cc0066b75/src/libraries/System.Private.Xml/src/System/Xml/Xsl/XsltOld/XsltCompileContext.cs#L985-L991

This is public API so it would be problematic to have this one wrong.

Reply:
XsltFunctionImpl has a public constructor which doesn't initialize that but at the same time is never referenced so I have `null!` initialized it to suppress